src/lightning_app/components/database/server.py

```diff
@@ -132,18 +138,44 @@ class CounterModel(SQLModel, table=True):
         """
         super().__init__(parallel=True, cloud_build_config=BuildConfig(["sqlmodel"]))
         self.db_filename = db_filename
+        self._root_folder = os.path.dirname(db_filename)
         self.debug = debug
+        self.store_interval = store_interval
         self._models = models if isinstance(models, list) else [models]
-        self.drive = None
+        self._store_thread = None
+        self._exit_event = None
+
+    def store_database(self):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            tmp_db_filename = os.path.join(tmpdir, os.path.basename(self.db_filename))
+
+            source = sqlite3.connect(self.db_filename)
+            dest = sqlite3.connect(tmp_db_filename)
+
+            source.backup(dest)
+
+            source.close()
+            dest.close()
+
+            drive = Drive("lit://database", component_name=self.name, root_folder=tmpdir)
+            drive.put(os.path.basename(tmp_db_filename))
+
+        print("Stored the database to the Drive.")
```

Comment:
Why `print` and not `logging.info`?

Reply:
we convert prints to info behind the scenes, this is what we adopted throughout